### ISSUE TITLE
Implement consistent create/save waiting message.

### ISSUE THREAD

**Issue by frankgiordano** [frankgiordano/PlatformTracker#62]
Introduce a progress bar for such operations. Even though these operations are quick and the progress bar will be barely seen, on a slow system it may take a while and best to show a progress bar so the end user knows the status and waits. 

Also take mouse cursor change and implement a wait for it too. This is already there but not implemented for all objects.

**Comment by frankgiordano** [comment]
done

**Comment by frankgiordano** [comment]
Reopening. Need to add wait progress bar for the linked resolutions to project screen.

**Comment by frankgiordano** [comment]
While working on the linked resolutions to project screen I specified $scope.loading = false; in the beginning of the controller code to make sure it is not showing (progress bar). I recall I first tried to execute the $scope.waiting(false) call to do the same at the top of the controller code but it did not work. I realised calling the method in the controller scope body not inside another method will not work if the method is defined first. So I will move the $scope.waiting() method on top and under it I will call it directly. I will refactor all accordingly.
